test/java/seedu/addressbook/utils/UtilsTest.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.utils;
+
+import seedu.addressbook.common.Utils;
+import static org.junit.Assert.*;
+import org.junit.Test;
+import java.util.Vector;
+
+public class UtilsTest {
+    
+    @Test
+    public void test_isAnyNull_allNotNull() {
+        assertTrue(Utils.isAnyNull('I', 2, 3, "don't", 5, "what", true, false));
+    }
+    
+    @Test
+    public void test_isAnyNull_someNull() {
+        assertTrue(Utils.isAnyNull(null, "am", 'I', true, 1, false, "doing", 7));
+    }
+    
+    @Test
+    public void test_isAnyNull_allNull() {
+        assertTrue(Utils.isAnyNull(null, null, null, null, null));
+    }
+    
+    @Test
+    public void test_elementsAreUnique_allUnique() {
+        Vector<Integer> testObjects = new Vector<Integer>();
+        for (int i = 0; i < 10; i++) {
+            testObjects.add(i);
+        }
```

Comment:
You can use `List<Integer> list = Arrays.asList(1, 1, 1, 2)` to create and initialize a list in one line.